pkg/resources/cloudcontroller.go

```diff
@@ -50,6 +50,9 @@ func ExternalCloudControllerFeatureSupported(dc *kubermaticv1.Datacenter, cloudS
 
 		fallthrough
 
+	case cloudSpec.AWS != nil:
+		fallthrough
```

Comment:
That's why it's using `fallthrough` and the vsphere `case` statement uses `kubermaticv1.ProviderType(cloudSpec.ProviderName)` to work for all providers, not just vsphere.

Reply:
I'm not sure that I like that at all. Someone might add something in between that would break this workflow. It's better to avoid using fallthrough and just combine those cases into a single case.